src/ServiceControl/CompositeViews/Messages/ScatterGatherApiMessageView.cs

```diff
@@ -1,17 +1,22 @@
 namespace ServiceControl.CompositeViews.Messages
 {
     using System.Collections.Generic;
+    using System.Linq;
     using Nancy;
 
-    public abstract class ScatterGatherApiMessageView<TInput> : ScatterGatherApi<TInput, MessagesView>
+    public abstract class ScatterGatherApiMessageView<TInput> : ScatterGatherApi<TInput, List<MessagesView>>
```

Comment:
By using IList here we could avoid the many ToLists but then we'd probably need to cast the IList to list in order to use the sort method which is only available on list :(